example/web/pubspec.yaml

```diff
@@ -0,0 +1,33 @@
+name: example
+description: A new Flutter project to quickly test flutter_launcher_icons.
+
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.7.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  cupertino_icons: ^0.1.3
+  flutter_launcher_icons:
+    path: ../..
+
+flutter_icons:
+  #  image_path: "assets/images/icon-128x128.png"
+  image_path_android: "assets/images/icon-710x599-android.png"
+  image_path_ios: "assets/images/icon-710x599-ios.png"
+  image_path_web: "assets/images/icon-710x599-web.png"
+  android: true # can specify file name here e.g. "ic_launcher"
+  ios: true # can specify file name here e.g. "My-Launcher-Icon"
+  web: true # Can't specify file name here at present.
+  adaptive_icon_background: "assets/images/christmas-background.png" # only available for Android 8.0 devices and above
+  adaptive_icon_foreground: "assets/images/icon-foreground-432x432.png" # only available for Android 8.0 devices and above
```

Comment:
```suggestion
  image_path_web: "assets/images/icon-710x599-web.png"
  web: true # Can't specify file name here at present.
```

Reply:
Okay! I can also get rid of the `android` and `ios` directories!